packages/Breadcrumbs/stories/Breadcrumbs.stories.mdx

```diff
@@ -0,0 +1,15 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+import { getStoryName } from "storybook/storyTree";
+import MDX from "./Breadcrumbs.mdx";
+import Breadcrumbs from "../src";
+
+export const params = {
+  options: {
+    isToolshown: true,
+    showPanel: false,
+  },
+};
+
+<Meta title={`${getStoryName("Breadcrumbs")}/Docs`} component={Breadcrumbs} parameters={params} />
+
+<MDX />
```

Comment:
No big deal, we can clean it up later, but since this file is so short, I'd put the contents of `Breadcrumbs.mdx` right here inline instead.

Reply:
I did it because I thought it's a pattern to have a `.stories.mdx` and a `.mdx` separately 😅